authorization/Cargo.toml

```diff
@@ -8,8 +8,6 @@ publish = false
 crate-type = ["cdylib", "rlib"]
 
 [features]
-default = ["export"]
-export = []
```

Comment:
Let's keep examples as simple as possible that's required for the area of scope they're demonstrating. I think we can leave these out.